draft-ietf-tls-ctls.md

```diff
@@ -817,27 +817,33 @@ This document requests that IANA change the name of entry 6 in the TLS
 HandshakeType Registry from "hello_retry_request_RESERVED" to
 "hello_retry_request", and set its Reference field to this document.
 
-## Reserved profiles
+## Profile ID Registry
 
 This document requests that IANA open a new registry entitled "Well-known
 cTLS Profile IDs", on the Transport Layer Security (TLS) Parameters page,
 with the following columns:
 
-* ID value: A sequence of 1-4 octets.
-* Template: A JSON object.
-* Note: An explanation or reference.
-
+* Value: A sequence of 1-4 octets.
+* Name: Name of the profile.
+* Description: Short description of the profile.
+* Change Controller: For Standards Track RFCs, list the "IESG".  For others,
+give thename of the responsible party.
+* Reference: Reference to the document or documents that specify the profile.
+ 
 The ID values of length 1 are subject to a "Standards Action" registry
 policy. Values of length 2 are subject to an "RFC Required" policy. Values
 of length 3 and 4 are subject to a "First Come First Served" policy. Values
 longer than 4 octets are not subject to registration and MUST NOT appear
 in this registry.
 
-The initial registry contents are:
+The initial registry content is:
+
+| Value     | Name        | Description                | Change Controller | Reference |
+|:=========:|:===========:|:==========================:|:====:|:=======:|
+| `[0x00]`  | base-cTLS   | Base cTLS 1.3-only Profile | IESG | RFCXXXX |
 
-| ID value  | Template           | Note          |
-|:=========:|:==================:|:=============:|
-| `[0x00]`  | `{"version": 772}` | cTLS 1.3-only |
```

Comment:
This change deletes the template, so there's no longer anything that explains the meaning of ID `[0x00]`.

Is there a reason not to include the full template in the registry?  Adding a layer of indirection is confusing to me, especially for FCFS registrations (length > 2).  Under this policy, these registrations could have a Reference that points to any "document", leading to interoperability challenges if that document changes or disappears.

Reply:
The profile id does more than just indicating a version number. Hence, a reader will always have to look at the specification that defines what the template does. Hence, I believe it is fine to have a reference to the spec included for the profile id.

If there is the worry that the referenced specification "disappears" then we should add that we want a stable reference there.

Reply:
Added text to the draft to get this point accross: "profile ID zero corresponds to a template that uses default settings for everything except the TLS version."